File: anonymous_015.py
def method(arr=[]):
    names=[]
    finalString=''
    for item in arr:
      names.append(item.get('name'))
    endName=names.pop()
    for name in names:
      finalString+=name+', '
    return endName if len(names)==0 else finalString.removesuffix(', ')+'&'+endName

File: anonymous_015.js
function method(arr) {
  let finalString = '';
  for (const i in arr) {
    finalString += arr[i].name;
    if (i < arr.length - 2) finalString += ', ';
    else if (i == arr.length - 2) finalString += '&';
  }
  return finalString;
}
